flink-java/src/main/java/org/apache/flink/api/java/functions/SelectByMaxFunction.java

```diff
@@ -36,7 +37,7 @@
 	 * is regarded in the reduce function. First index has highest priority and last index has
 	 * least priority.
 	 */
-	public SelectByMaxFunction(TupleTypeInfo<T> type, int... fields) {
+	public SelectByMaxFunction(TypeInformation<T> type, int... fields) {
```

Comment:
Done. Since we have added new scala class these changes are unnecessary. Thanks for pointing it out.